### PR TITLE
[Taxon] Add missing locale error quering taxon details

### PR DESCRIPTION
**Pull request by lchrusciel**

### CODE REVIEW

File: src/Controller/Taxon/ShowTaxonDetailsAction.php [hunk 1]
```diff
@@ -44,6 +46,10 @@ public function __invoke(Request $request): Response
         $code = $request->attributes->get('code');
         $locale = $request->query->get('locale');
 
+        if (null === $locale) {
+            throw new BadRequestHttpException('You haven\'t provide a locale!');
```
Comment (venyii) [inline src/Controller/Taxon/ShowTaxonDetailsAction.php]: `haven't provided` or `didn't provide`

**Comment by zmiter** [comment]
IMHO, maybe it's better to inject $fallbackLocale variable in the same way it's done in ShowTaxonTreeAction?
I was about to send a PR in that time found out an issue has been already posted here =)

**Comment by mamazu** [comment]
I don't think that it should throw an exception this would be more of a thing for validation or as suggested use a default locale.

File: tests/Responses/Expected/taxon/missing_locale_exception_response.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+    "code": 400,
+    "message": "You haven't provide a locale!"
```
Comment (CoderMaggie) [inline tests/Responses/Expected/taxon/missing_locale_exception_response.json]: A bit of grammar change: "`You did not provide a locale!"`

```suggestion
    "message": "You did not provide a locale!"
```

**Comment by lchrusciel** [comment]
It is not a good solution. Closing for now.

**Pull request closed without merging**